src/stories/index.stories.js

```diff
@@ -65,7 +68,40 @@ storiesOf('Basic', module)
             template: `<Identicon :address="address"/>`,
         };
     })
-    .add('Input', () => {
+    .add('IdenticonContact', () => {
+        const address = text('address', 'NQ07 0000 00000000 0000 0000 0000 0000 0000');
+        const name = 'Nimiq';
+        return {
+            components: {IdenticonContact},
+            data() {
+                return { address, name };
+            },
+            template: `<IdenticonContact :address="address" :name="name"/>`,
+        };
+    })    
+    .add('IdenticonContact (empty)', () => {
```

Comment:
I don't think it's necessary to create a story just for that.
A knob for the address is defined anyways.
Might want to define one for the name also.